Source/JavaScriptCore/bytecode/BytecodeUseDef.h

```diff
@@ -32,17 +32,17 @@
 
 namespace JSC {
 
-void computeUsesForBytecodeIndexImpl(VirtualRegister, const JSInstruction*, Checkpoint, const ScopedLambda<void(VirtualRegister)>&);
+void computeUsesForBytecodeIndexImpl(const JSInstruction*, Checkpoint, const ScopedLambda<void(VirtualRegister)>&);
 void computeDefsForBytecodeIndexImpl(unsigned, const JSInstruction*, Checkpoint, const ScopedLambda<void(VirtualRegister)>&);
 
 template<typename Block, typename Functor>
 void computeUsesForBytecodeIndex(Block* codeBlock, const JSInstruction* instruction, Checkpoint checkpoint, const Functor& functor)
 {
     OpcodeID opcodeID = instruction->opcodeID();
-    if (opcodeID != op_enter && (codeBlock->wasCompiledWithDebuggingOpcodes() || codeBlock->usesCallEval()) && codeBlock->scopeRegister().isValid())
```

Comment:
This appears to be the only / last real usage of `usesCallEval()` and `setUsesCallEval()`. I wonder if we can remove those completely?

Reply:
Oh, wow! We should remove it.